src/Microsoft.ML.StandardTrainers/Microsoft.ML.StandardTrainers.csproj

```diff
@@ -1,7 +1,8 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netstandard2.0</TargetFramework>
+    <TargetFramework Condition="'$(UseIntrinsics)' != 'true'">netstandard2.0</TargetFramework>
+    <TargetFrameworks Condition="'$(UseIntrinsics)' == 'true'">netstandard2.0;netcoreapp3.0</TargetFrameworks>
```

Comment:
The NuGet package is not going to work the way this is currently written. See [https://github.com/dotnet/machinelearning/issues/534](https://github.com/dotnet/machinelearning/issues/534) for more information. Basically, once you introduce a single assembly in `lib\netcoreapp3.0` of the NuGet package, ALL the assemblies need to be in the `lib\netcoreapp3.0` folder of the package.

It would probably be easiest if the new AVX/FMA were put in the CpuMath assembly/package instead.